### PR TITLE
[terra-outline-table-view] Creating outline table view private package

### PR DESCRIPTION
**Pull request by BenBoersma**
### Summary
 Creating the basic directories for the new outline table view component including a starter test and doc

Closes # UXPLATFORM-6829

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-outline-table-view/NOTICE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Copyright 2017 - 2019 Cerner Innovation, Inc.
```
Comment (supreethmr) [inline packages/terra-outline-table-view/NOTICE:1]: should this be latest 
```suggestion
Copyright 2022 - 2023 Cerner Innovation, Inc.
```
OR 2021 -2022

File: packages/terra-outline-table-view/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+| react-intl | ^2.8.0 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## LICENSE
+
+Copyright 2017 - 2019 Cerner Innovation, Inc.
```
Comment (supreethmr) [inline packages/terra-outline-table-view/README.md:31]: same here 2022

File: packages/terra-outline-table-view/package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
```
Comment (supreethmr) [inline packages/terra-outline-table-view/package.json:29]: Readme.md file has one more peer dependency `react-intl`. it is missing here :)

File: packages/terra-outline-table-view/NOTICE (hunk 1 above)
Comment (BenBoersma) [reply packages/terra-outline-table-view/NOTICE]: Fixed here https://github.com/cerner/terra-core/pull/3651/commits/639048e63b7776165b0ce16a2af9d6ce67405738#diff-9b46e6ad6c1ee4e98a1dc86544215549059aabed30e57c721975b71d9afe2700R1

File: packages/terra-outline-table-view/README.md (hunk 2 above)
Comment (BenBoersma) [reply packages/terra-outline-table-view/README.md]: Fixed here https://github.com/cerner/terra-core/pull/3651/commits/639048e63b7776165b0ce16a2af9d6ce67405738#diff-3ec8140833b830c14db1405b5b40917b256df980202f3e9a00365cca77023fd6R31

File: packages/terra-outline-table-view/package.json (hunk 3 above)
Comment (BenBoersma) [reply packages/terra-outline-table-view/package.json]: added here https://github.com/cerner/terra-core/pull/3651/commits/639048e63b7776165b0ce16a2af9d6ce67405738#diff-5f92e8897649e36af2fd07d1abafe46a89d61e8f79446285b4a4d69ef2a260ffR30

**Review by supreethmr — APPROVED** [review]

File: packages/terra-outline-table-view/.npmignore [hunk 4]
```diff
@@ -0,0 +1,9 @@
+*.log
+target
+reports
+tests/**/__snapshots__/
+aggregated-translations
+dev-site-config
+screenshots
+errorScreenshots
+generated-themes
```
Comment (sdadn) [inline packages/terra-outline-table-view/.npmignore:9]: nit: new line

**Review by sdadn — APPROVED** [review]

File: packages/terra-outline-table-view/.npmignore (hunk 4 above)
Comment (BenBoersma) [reply packages/terra-outline-table-view/.npmignore]: Fixed here https://github.com/cerner/terra-core/pull/3651/commits/e3a410ecfbde59ea3b1ac736e1239315a0e96814

**Pull request merged**